flowmachine/flowmachine/core/init.py

```diff
@@ -185,15 +191,18 @@ def connect(
     return Query.connection
 
 
-def _init_logging(log_level, log_file):
+def _init_logging(log_level, write_log_file):
     """
 
     Parameters
     ----------
     log_level : str
         Level to emit logs at
-    log_file : str
-        Path to file to write logs to
+    write_log_file : bool, default False
```

Comment:
`write_log_file` has no default in `_init_logging`

Reply:
Good catch. The default value is set in `connect()` and then passed in here, so we don't need a default here. I'll remove it from the docstring.